### ISSUE TITLE
[Bug]: The Wiki Loves Monuments image request is still displayed after uploading

### ISSUE THREAD

**Issue by OpenGreenStreet** [commons-app/apps-android-commons#5805]
### Summary

The Wiki Loves Monuments image request is still displayed after uploading

### Steps to reproduce

1. Uploading the image
2. Return to "nearby"
3. The picture request can still be seen, or it is not recognizable that a picture was uploaded by this user shortly before on this topic

### Expected behaviour

The Wiki Loves Monuments image request is no longer displayed after uploading or it is symbolized that this user has already uploaded an image on this topic shortly before.

### Actual behaviour

The Wiki Loves Monuments image request is still displayed after upload or it is not recognizable that this user has already uploaded an image on this topic shortly before.

### Device name

Google Pixel Pro 7

### Android version

Android 14

### Commons app version

5.0.2~05ffd123e 

### Screen-shots

* https://commons.wikimedia.org/wiki/File:20240902_xl_1813-WLM-Bilderwunsch_wird_nach_Upload_weiterhin_angezeigt_2.png

* https://commons.wikimedia.org/wiki/File:20240902_xl_1813-WLM-Bilderwunsch_wird_nach_Upload_weiterhin_angezeigt.png

### Would you like to work on the issue?

None

**Comment by nicolas-raoul** [comment]
Even after restarting the phone/app?

**Comment by OpenGreenStreet** [comment]
@nicolas-raoul ,

It remains unchanged even after restarting the smartphone. And even after closing the app and restarting the smartphone again, it remains the same.

In principle, I think that's fine: I might want to publish a photo of the same object from a different perspective.

But if I do, then it should be clear - for example with a slightly different icon - that I have already worked on the object.

**Comment by nicolas-raoul** [comment]
What you describe happens only for Wiki Loves Monuments, not for other ("normal") pins, right?

It might be intended, as WLM is mostly about capturing all monuments, not only those with no P18 image yet.

> it should be clear - for example with a slightly different icon - that I have already worked on the object.

Great idea for an enhancement, I edit the description to reflect that.

**Comment by OpenGreenStreet** [comment]
@nicolas-raoul ,

Yes, exactly, what I am describing here only concerns pins for Wiki Loves Monuments, not “normal” pins.

(Just by the way, even with “normal” pins I experience (very rarely) that some appear and “disappear” again. This is also one of the reasons why I would love to have a special “Update”/“Download here” button).

**Comment by OpenGreenStreet** [comment]
„It might be intended, as WLM is mostly about capturing all monuments, not only those with no P18 image yet.“
@nicolas-raoul 
PS: I suspect that too. But in places with a lot of monuments, it gets confusing.

**Comment by nicolas-raoul** [comment]
About "Update" I created https://github.com/commons-app/apps-android-commons/issues/5806
